ui/app/common/photo-capture/directives/capturePhoto.js

```diff
@@ -130,7 +134,18 @@ angular.module('bahmni.common.photoCapture')
             };
 
             scope.uploadImage = function () {
-                if (this.files && this.files[0]) {
+                if (this.files && this.files[0] && this.files[0].type) {
+                    var fileType = this.files[0].type;
+                    if (!fileType.startsWith('image/')) {
+                        uploadConfirmImageButton.prop('disabled', true);
+                        alert('File uploaded must be an image');
```

Comment:
These are user messages, and should not be hardcoded in english. Need to be translated.

Reply:
Sure Gurpreet. This current PR is a duplicate one. I will complete the above comment in a separate card. So will create a separate card and will raise PR accordingly. So closing this PR.